mkosi/config.py

```diff
@@ -527,6 +539,29 @@ def config_parse_profile(value: Optional[str], old: Optional[int] = None) -> Opt
     return value
 
 
+def parse_drive(value: str) -> ConfigDrive:
+    parts = value.split(":")
```

Comment:
can qemu options relevant to drives contain colons? If so 
```suggestion
    parts = value.split(":", maxsplit=3)
```